browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1456,6 +1456,24 @@ Are you sure you want to do this?
       <message name="IDS_BLOCK_COOKIES_OPTION_3" desc="Title for shields option.">
         Allow all cookies
       </message>
+      <message name="IDS_REQUEST_OTR" desc="Title for shields option.">
+        Request Off-The-Record
+      </message>
+      <message name="IDS_REQUEST_OTR_TEXT" desc="Text for shields option.">
+        Go Off-The-Record when visiting sensitive sites
+      </message>
+      <message name="IDS_REQUEST_OTR_SUMMARY" desc="Summary for Request Off-The-Record option.">
+        Disables browser history and local storage when sites request it
+      </message>
+      <message name="IDS_REQUEST_OTR_OPTION_1" desc="Title for shields option.">
+        Always go off-the-record
+      </message>
+      <message name="IDS_REQUEST_OTR_OPTION_2" desc="Title for shields option.">
+        Ask before going off-the-record (standard)
+      </message>
+      <message name="IDS_REQUEST_OTR_OPTION_3" desc="Title for shields option.">
+        Never go off-the-record
```

Comment:
Sometimes Off-The-Record is capitalized and sometimes it isn't. Is it meant to be inconsistent?